configdist.py

```diff
@@ -31,7 +31,8 @@
 TWILIO_TEST_NUMBER = ""
 
 SECRET_KEY = 'This is not secret you must change it'
-HOST_NAME = 'http://localhost:5000'
+PREFERRED_URL_SCHEME = "https"
```

Comment:
The default here would be `http` because by default, you would be using the built-in development server that does not use HTTPS.